components/automate-ui/src/app/entities/desktop/desktop.requests.ts

```diff
@@ -77,6 +77,20 @@ export interface RespDesktop {
   deprecations_count: number;
   chef_version: string;
   domain: string;
```

Comment:
Need to include more data fields now that you can choose to view them in desktop list.